### DISCUSSION TITLE
Pocketbase integration

### DISCUSSION THREAD

**Discussion by ruvido** [nocodb/nocodb#4340] [Features]
Pocketbase is a self-hosted sqlite implementation in a single go binary. Is it possible to view and edit the pocketbase data and how?
FYI: https://pocketbase.io

**Comment by o1lab** [comment]
NocoDB connects to any SQLite and transforms into a collaborative spreadsheet. So, yes - you can connect to your SQLite from pocketbase via NocoDB
